crates/node/src/lib.rs

```diff
@@ -476,6 +480,13 @@ impl Node {
             });
         }
 
+        let payload = if !aliases.is_empty() {
+            self.substitute_aliases_in_payload(context_id, payload, &aliases)
+                .await?
+        } else {
+            payload
+        };
```

Comment:
no need to redeclare, since the type didn't change

```suggestion
        if !aliases.is_empty() {
            payload = self.substitute_aliases_in_payload(context_id, payload, &aliases)
                .await?
        }
```